catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/ResourceOperations.java

```diff
@@ -664,7 +664,11 @@ protected ResourceInfo getResourceInfo(ResourceRequest resourceRequest, String s
         return new ResourceInfo(metacard, resourceUri);
     }
 
-    private Query anyTag(Query query) {
+    private Query anyTag(Query query, String site, boolean isEnterprise) {
+        if (isEnterprise || !getId().equals(site)) {
```

Comment:
doesn't an enterprise search include the local catalog? Just thinking that in the enterprise case it would work for all sources except local.